### PR TITLE
Support latest version of SlackRubyBot by applying instrumentation to message hook

### PR DESCRIPTION
**Pull request by kstole**
For now, I think we should just apply the instrumentation to the built in message hook and then when the hooks mechanism in SlackRubyBot is updated to use the `on` method, we can look at changing this. @dblock thoughts?

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -2,6 +2,7 @@ source 'http://rubygems.org'
 
 gemspec
 
+gem 'celluloid-io', require: %w(celluloid/current celluloid/io)
```
Comment (dblock) [inline Gemfile]: These should probably be inside a test or development group(s).

File: newrelic-slack-ruby-bot.gemspec [hunk 2]
```diff
@@ -14,5 +14,5 @@ Gem::Specification.new do |s|
   s.licenses = ['MIT']
   s.summary = 'NewRelic instrumentation for slack-ruby-bot.'
   s.add_dependency 'newrelic_rpm'
-  s.add_dependency 'slack-ruby-bot', '<= 0.7.0'
+  s.add_dependency 'slack-ruby-bot', '> 0.7.0'
```
Comment (dblock) [inline newrelic-slack-ruby-bot.gemspec]: This should be >= the exact version with which it works, so if 0.8.0 was the first one where this change shipped, it should be >= 0.8.0.

**Review by dblock — APPROVED** [review]
Looks great, needs a rebase + see some of my comments.

**Comment by kstole** [comment]
Shall we merge this and then create a new release?

**Comment by dblock** [comment]
Yup, merged, and thanks!

**Pull request merged**